tests/data_context/test_data_context_variables.py

```diff
@@ -99,6 +106,20 @@ def cloud_data_context_variables(
     )
 
 
+@pytest.fixture
+def file_data_context(
+    tmp_path, data_context_config: DataContextConfig
+) -> FileDataContext:
+    project_path = tmp_path / "file_data_context"
+    project_path.mkdir()
+    project_path = str(project_path)
+    context_root_dir = os.path.join(project_path, "great_expectations")
```

Comment:
I would keep using `pathlib` here instead of `os`.

https://treyhunner.com/2018/12/why-you-should-be-using-pathlib/
https://treyhunner.com/2019/01/no-really-pathlib-is-great/

```suggestion
    context_root_dir = project_path / "great_expectations"
```